src/components/viewLayout/viewLayout.tsx

```diff
@@ -48,7 +48,12 @@ const AppLayout: React.FC<AppLayoutProps> = ({
   const Header = (
     <Masthead>
       <MastheadToggle>
-        <Button variant="plain" onClick={() => setSidebarOpen(!sidebarOpen)} aria-label="Global navigation">
+        <Button
+          variant="plain"
+          onClick={() => setSidebarOpen(!sidebarOpen)}
+          aria-label="Global navigation"
+          ouiaId="global-navigation"
```

Comment:
This is button without text content and I needed _something_ to find it in unit test.

Reply:
You are right. I wouldn't think of finding element by **aria** using getBy**Role**.

I would like to keep ouiaId in case we ever want to interact with that element from Camayoc, and to reinforce the idea that ouiaId are important in this project and we should overuse them than underuse them.

Reply:
fine with that!